stablehlo/dialect/VhloAttrs.td

```diff
@@ -56,6 +56,34 @@ class VHLO_AttrDef<string name,
   }];
 }
 
+//===----------------------------------------------------------------------===//
+// Wrapped Attributes
+//===----------------------------------------------------------------------===//
+
+def VHLO_AttrWrap : VHLO_AttrDef<"Wrapped"> {
```

Comment:
I probably won't have time to review this in detail today, but I wanted to ask a high-level question that we could already iterate on. Given that both the attributes and the types are currently split between forking and not forking, how about we consistently fork everything? How much additional work would that be?

Reply:
Not too much. I agree this will be helpful. We'll have to be a bit careful on forking `FloatAttr` and `IntegerAttr` to make sure we maintain proper textual parsing/printing of numerics, but otherwise I think it's fairly simple. I'll get to that addition.

Reply:
Forked all attributes. Will do the same for types soon. I still wrapped integer, but with an integer type that validates the underlying value and will likely do the same for float. It pushes a bit of code into bytecode, but I think it works out better for printing/parsing/memory storage of values.